### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by pboling**
I'm not sure what should go in this file, and that is precisely the problem.

@BobbyMcWho @suprnova32 who is the current maintainer?  Need guidance on disclosing a security vulnerability.

### CODE REVIEW

**Comment by BobbyMcWho** [comment]
@pboling I don't think there's anyone that actively claims it. I have merge rights over pretty much everything in omniauth, but I don't have rubygems access to the gem. https://rubygems.org/gems/omniauth-saml

I don't particularly want to add another ruby responsibility as I'm mostly in Elixir land these days, though an occasional review/merge is okay, I'm not sure who should own a vulnerability report.

**Comment by suprnova32** [comment]
There hasn't been a new version of this gem in over 2 years. The only supported version is the latest one. If there are security vulnerabilities related to the code within this gem, creating an issue here to let the maintainers know you have found a vulnerability and how to best contact you is the way to go. We can then take action from there, specially if the vulnerability is dangerous and cannot be discussed in public.

If the vulnerability is related to libraries this gem depends on, the report should be made on those projects. Once they have been updated, we can update this gem to include the newest versions.

**Comment by pboling** [comment]
It is a vulnerability in a dependency of this gem, which has not yet been disclosed, but which will affect users of this gem, and which will require upgrades.  The discloser is attempting to discuss it with maintainers of this gem, which is how I got looped in, as an adjacent omniauth-* plugin maintainer, but I cannot find your email address, @suprnova32.  I can loop you into the email chain if you would like to email me.  I can't find your email listed anywhere online.  You can find my email address on my Github profile, or at https://peterboling.com/contact

Separately, this project needs a documented security policy.

> The only supported version is the latest one. If there are security vulnerabilities related to the code within this gem, creating an issue here to let the maintainers know you have found a vulnerability and how to best contact you is the way to go. We can then take action from there, specially if the vulnerability is dangerous and cannot be discussed in public.

> If the vulnerability is related to libraries this gem depends on, the report should be made on those projects. Once they have been updated, we can update this gem to include the newest versions.

Perhaps we can just slot that into the README?

**Comment by pboling** [comment]
The vulnerability will be disclosed tomorrow.

**Comment by bufferoverflow** [comment]
I think we should rather enable the built-in functionality:

image

I can also publish this gem, but as we do not have a lot of activity there was no need to publish a new version.

I have enabled **Private vulnerability reporting** now.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (bufferoverflow) [inline SECURITY.md:13]:
```suggestion
|   2.1.x  | :white_check_mark: |
| < 2.1.x  | :x:               |
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
```
Comment (bufferoverflow) [inline SECURITY.md:21]:
```suggestion
You can report a vulnerability via https://github.com/omniauth/omniauth-saml/security.
```

**Comment by bufferoverflow** [comment]
@pboling Could you please rebase and then we can merge this.

btw. new release is published https://rubygems.org/gems/omniauth-saml

**Comment by pboling** [comment]
@bufferoverflow done!

**Review by bufferoverflow — APPROVED** [review]

**Pull request merged**